client/flutter/lib/pages/travel_advice.dart

```diff
@@ -1,4 +1,3 @@
-import 'package:WHOFlutter/generated/l10n.dart';
```

Comment:
looks like these were removed as unused imports, yes?

Reply:
Yes, OP describes it